### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by gwezerek**

### CODE REVIEW

**Comment by gwezerek** [comment]
@gregshap This is ready for your stamp of approval. Still couldn't test Facebook shares because beta's behind a pw so 401 err.

File: html/rankings/growth-predictions-list.html [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+  <header class="news__header">
+    <h3 class="news__head">Growth Projections, 2013</h3>
+    <p class="news__p"><span class="news__date">May 7, 2015</span></p>
+  </header>
+
+  <table class="news__table">
+    <thead>
+      <tr class="news__tr">
+        <th class="news__th">Country</th>
+        <th class="news__th">Project Annual Growth Through 2023 (%)</th>
```
Comment (gregshap) [inline html/rankings/growth-predictions-list.html]: just noting from our convo, should be projectED annual growth

File: html/rankings/growth-predictions-list.html [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+  <ul class="sidebar__list">
+    <li class="sidebar__item"><a href="/media/growth_projections/country_rankings_and_projections_2013.csv">&raquo; Download 2013 data and growth projections</a></li>
+    <li class="sidebar__item"><a href="rankings/country/download/">&raquo; Download all years data</a></li>
+  </ul>
+</div>
+
+<article class="news__article">
+
+  <header class="news__header">
+    <h3 class="news__head">Growth Projections, 2013</h3>
```
Comment (gregshap) [inline html/rankings/growth-predictions-list.html]: :speak_no_evil: not a bug, but this title makes it sound like the story is 2 years old. Would it be better if we said something like: "Growth Projects based on 2013 Annual Exports?" Or is that only better in my head?

Comment (gwezerek) [inline html/rankings/growth-predictions-list.html]: Nope, that makes total sense!

**Pull request merged**